doc/CHECKERS.md

```diff
@@ -1 +1,2 @@
+# Checkers in CVE Binary Tool
```

Comment:
Does adding this not break the way the link works when the files are parsed?  (Files with just a filename in them are effectively symbolic links)